server/upload/cmd/uploadserver/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/google/prog-edu-assistant/uploadserver"
+)
+
+var (
+	port        = flag.Int("port", 8000, "The port to serve HTTP/S.")
+	useHTTPS    = flag.Bool("use_https", false, "If true, use HTTPS instead of HTTP.")
+	sslCertFile = flag.String("ssl_cert_file", "localhost.crt",
+		"The path to the signed SSL server certificate.")
+	sslKeyFile = flag.String("ssl_key_file", "localhost.key",
+		"The path to the SSL server key.")
+	uploadDir = flag.String("upload_dir", "uploads", "The directory to write uploaded notebooks.")
```

Comment:
"=" are not indented at the same level.
Did you run `go fmt`?

Reply:
Yes, I did run gofmt. The equal signs are not aligned because there is a wrapping line separating them. Gofmt only tries to align equal signs if the are in adjacent lines.